tests/test_playbooks/smart_class_parameter.yml

```diff
@@ -0,0 +1,558 @@
+---
```

Comment:
Can you add a comment that the (which?) `ntp` puppet module needs to be present for re-recording this?

Like we do in https://github.com/theforeman/foreman-ansible-modules/blob/923f37f22af15f0b883e286fbb6a2ce1ce66a9c6/tests/test_playbooks/config_group.yml#L2-L5  or https://github.com/theforeman/foreman-ansible-modules/blob/923f37f22af15f0b883e286fbb6a2ce1ce66a9c6/tests/test_playbooks/hostgroup.yml#L2-L8

Or isn't that needed?

Reply:
Yes it's needed. Added comment.